src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -108,6 +118,16 @@ protected GenInputsAbstract(
   @Option("File that lists methods under test")
   public static Path methodlist = null;
 
+  /**
+   * Automatically add dependencies to the classes under test. For any classes, methods, and
+   * constructors specified by the user, this option adds their argument classes. (The user
+   * specifies them using the {@code --testjar}, {@code --classlist}, {@code --testclass}, and
+   * {@code --methodlist} options.) Recommended to use with {@code --require-covered-classes}
```

Comment:
It`s the second one. This suggestion was added because we can end up with more tests for dependencies than for code we actually wanted to test. The logic behind it is dependencies should be tested separately, doesn`t matter using Randoop or hand-written tests.